### PR TITLE
Document validity of fields of VideoPlayerController

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: packages/video_player/lib/video_player.dart [hunk 1]
```diff
@@ -32,13 +32,34 @@ class DurationRange {
...
+
+  /// True iff the video is looping.
   final bool isLooping;
+
+  /// The current volume of the playback.
   final double volume;
+
+  /// A description of the error.
+  ///
+  /// If [isErroneous] is false this is [null].
```
Comment (mravn-google) [inline packages/video_player/lib/video_player.dart]: Could we change `isErroneous` to `hasError` as in the camera plugin?
(Would be a breaking change -> version 0.2.0, I guess)

File: packages/video_player/lib/video_player.dart [hunk 2]
```diff
@@ -32,13 +32,34 @@ class DurationRange {
...
+  /// True if the video is playing. False if it's paused.
   final bool isPlaying;
+
+  /// True iff the video is looping.
   final bool isLooping;
+
+  /// The current volume of the playback.
   final double volume;
+
+  /// A description of the error.
```
Comment (mravn-google) [inline packages/video_player/lib/video_player.dart]: ... if present.

File: packages/video_player/lib/video_player.dart [hunk 3]
```diff
@@ -32,13 +32,34 @@ class DurationRange {
 }
 
 class VideoPlayerValue {
+  /// The total duration of the video.
+  ///
+  /// Before the [VideoPlayerController] is initialized this is null.
```
Comment (mravn-google) [inline packages/video_player/lib/video_player.dart]: [nit] Maybe `/// Is null iff [initialized] as false.`

File: packages/video_player/lib/video_player.dart [hunk 4]
```diff
@@ -32,13 +32,34 @@ class DurationRange {
...
+  /// The current playback position.
   final Duration position;
+
+  /// The currently buffered ranges.
   final List<DurationRange> buffered;
+
+  /// True if the video is playing. False if it's paused.
   final bool isPlaying;
+
+  /// True iff the video is looping.
```
Comment (mravn-google) [inline packages/video_player/lib/video_player.dart]: Reconsider `iff`.

File: packages/video_player/lib/video_player.dart (hunk 1 above)
Comment (sigurdm) [reply packages/video_player/lib/video_player.dart]: Done

File: packages/video_player/lib/video_player.dart (hunk 2 above)
Comment (sigurdm) [reply packages/video_player/lib/video_player.dart]: Done

File: packages/video_player/lib/video_player.dart (hunk 4 above)
Comment (sigurdm) [reply packages/video_player/lib/video_player.dart]: Done

File: packages/video_player/lib/video_player.dart (hunk 3 above)
Comment (sigurdm) [reply packages/video_player/lib/video_player.dart]: Went for
```
  /// Is null when [initialized] is false.
```

File: packages/video_player/lib/video_player.dart [hunk 5]
```diff
@@ -32,13 +32,34 @@ class DurationRange {
 }
 
 class VideoPlayerValue {
```
Comment (Hixie) [inline packages/video_player/lib/video_player.dart]: class docs?

**Comment by Hixie** [comment]
@sigurdm what's the status of this PR?

**Comment by sigurdm** [comment]
Thanks for the ping. I had forgot about this!

File: packages/video_player/lib/video_player.dart (hunk 5 above)
Comment (sigurdm) [reply packages/video_player/lib/video_player.dart]: Done

**Pull request merged**